tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
It's usually common to delete the comments after a `tsc --init`. These are just helpers for configuring your setup.